$NetBSD: patch-Makefile,v 1.1 2012/08/19 07:01:09 marino Exp $

LDFLAGS is overriding pkgsrc, so add rpath to ${PREFIX}/lib to find libtclXX.so

--- Makefile.orig	2012-08-18 13:33:02.582480000 +0000
+++ Makefile
@@ -5,7 +5,7 @@ PIC = -fPIC
 INCLUDES = -I${PREFIX}/include/tcl -I$(MYSQLINCDIR)
 CFLAGS += -Wall $(PIC) $(INCLUDES)
 
-LDFLAGS =  -lz $(MYSQLLIBDIR)/libmysqlclient.la -L$(PREFIX)/lib -ltcl
+LDFLAGS =  -lz $(MYSQLLIBDIR)/libmysqlclient.la -L$(PREFIX)/lib -R${PREFIX}/lib -ltcl
 
 all:	fbsql.la
 
